tests/unit/v2/cache/testMemoryBackend.js

```diff
@@ -103,4 +103,27 @@ describe('Test cache memory backend', () => {
             assert(res, true);
         });
     });
+
+    it('should update the account size counter', async () => {
+        await Promise.all(testValues.map(event => mem.updateCounters(event)));
```

Comment:
I think you can just pass update counters to the map function.
```suggestion
        await Promise.all(testValues.map(mem.updateCounters));
```

Reply:
`updateCounters` would lose its reference to the correct this if not wrapped or prebound with `mem.updateCounters.bind(mem)`